### PR TITLE
[MS2] 1. Analytics

### PR DESCRIPTION
**Pull request by WiNloSt**
Epic: #39246 Milestone 2
[Analytics spec](https://www.notion.so/metabase/Let-white-labeling-customers-customize-illustrations-24e120498d5f414aaf52ddd0de9e20fc?pvs=4#5f222393a48f4193aff63e4f8e06d69d)

The stats typo was also [cherry-picked to 49 already](https://github.com/metabase/metabase/pull/40721)

### Description

Add new settings and remove an old setting from/to anonymous analytics

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: enterprise/frontend/src/metabase-enterprise/settings/selectors.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export function getLoginPageIllustration(
         isDefault: true,
       };
 
-    case "no-illustration":
+    case "none":
```
Comment (WiNloSt) [inline enterprise/frontend/src/metabase-enterprise/settings/selectors.ts:58]: adjust the value to match the value in [analytics](https://www.notion.so/metabase/Let-white-labeling-customers-customize-illustrations-24e120498d5f414aaf52ddd0de9e20fc?pvs=4#0f9388c8dac74020970f29f485f80989) for simplicity

File: docs/installation-and-operation/serialization.md [hunk 2]
```diff
@@ -252,7 +252,6 @@ site-name
 application-font-files
 loading-message
 report-timezone
-show-lighthouse-illustration
```
Comment (WiNloSt) [inline docs/installation-and-operation/serialization.md:255]: Replaced by `login-page-illustration`

File: src/metabase/analytics/stats.clj [hunk 3]
```diff
@@ -126,31 +126,34 @@
...
+   :slack_configured                     (slack/slack-configured?)
+   :sso_configured                       (google/google-auth-enabled)
+   :instance_started                     (snowplow/instance-creation)
+   :has_sample_data                      (t2/exists? Database, :is_sample true)
+   :enable_embedding                     (embed.settings/enable-embedding)
+   :embedding_app_origin_set             (boolean (embed.settings/embedding-app-origin))
+   :appearance_site_name                 (not= (public-settings/site-name) "Metabase")
+   :appearance_help_link                 (public-settings/help-link)
+   :appearance_logo                      (not= (public-settings/application-logo-url) "app/assets/img/logo.svg")
+   :appearance_favicon                   (not= (public-settings/application-favicon-url) "app/assets/img/favicon.ico")
```
Comment (WiNloSt) [inline src/metabase/analytics/stats.clj:147]: Use hide whitespace to review this file to see the actual diff
image

File: src/metabase/analytics/stats.clj [hunk 4]
```diff
@@ -126,31 +126,34 @@
...
+   :appearance_loading_message           (not= (public-settings/loading-message) :doing-science)
+   :appearance_metabot_greeting          (not (public-settings/show-metabot))
+   :appearance_login_page_illustration   (public-settings/login-page-illustration)
```
Comment (WiNloSt) [inline src/metabase/analytics/stats.clj:150]: These are the new settings

**Review by deniskaber — APPROVED** [review]

**Review by sansolovyov — APPROVED** [review]
👍

**Review by npretto — APPROVED** [review]

**Pull request merged**